src/mqt/predictor/reward.py

```diff
@@ -69,3 +70,90 @@ def calc_qubit_index(qargs: list[Qubit], qregs: list[QuantumRegister], index: in
             return qubit_index
     error_msg = f"Global qubit index for local qubit {index} index not found."
     raise ValueError(error_msg)
+
+
+def expected_success_probability(qc: QuantumCircuit, device: Device, precision: int = 10) -> float:
+    """Calculates the expected success probability of a given quantum circuit on a given device.
+
+    Args:
```

Comment:
Please use "Arguments" instead of "Args". Last week, I unified the docstrings and introduced that style.